torch/csrc/jit/script/module.h

```diff
@@ -633,11 +633,13 @@ struct Module {
       parameter_remap[param] = curr->parameter_slot(param.name());
     }
     for (auto& attr : get_attributes()) {
-      if (!attr.type()->isSubtypeOf(TensorType::get())) {
-        continue;
+      if (attr.type()->isSubtypeOf(TensorType::get())) {
```

Comment:
Pretty sure register_buffer calls register_attribute. Do we need to call register buffer here?